docs/developers/build.md

```diff
@@ -54,13 +54,7 @@ chosen:
 ### OpenIndiana
 
 On OpenIndiana, the `build-essential` package includes the GCC compilers
-and other tools required to build illumos.  In addition, for now, we
-should install the older Python 3.5 package for a complete build.
-
-```
-sudo pkg install build-essential \
-    runtime/python-35
-```
```

Comment:
I think we should leave the command to install the package here, just with the amended list of packages?

Reply:
Agreed.   Rebased and changed as requested.